### DISCUSSION TITLE
How to provide whistleblowers the possibility to optionally provide their identity?

### DISCUSSION THREAD

**Discussion by pcinacio** [globaleaks/globaleaks-whistleblowing-software#3867] [Q&A]
Good evening.
I want to modify the provided questionnaire to "allow" the whistleblowers to either identify themselves or remain anonymous. I believe I saw a solution using GlobaLeaks where the questionnaire began with 2 sections. 
The first section provided the whistleblower with the choice to identify themselves or not, and the second section was the actual questionnaire. Is there any place where I can find documentation on how to create in this manner? Thank you.

**Comment by evilaliv3** [comment]
Try to include the template questions "Would you like to tell us who you are?". This is probably what you are looking for.
